src/lib/MedStickerEncryptionCharlie.js

```diff
@@ -52,28 +52,30 @@ export function splitKeys(arrayBuffer) {
 }
 
 /**
- * @param key {CryptoKey}
- * @param iv {Uint8Array}
+ * @param key {ArrayBuffer}
+ * @param iv {ArrayBuffer}
  * @param toEncryptBytes {ArrayBuffer}
  * @returns {Promise<ArrayBuffer>}
  */
 export async function encrypt(key, iv, toEncryptBytes) {
   try {
-    return await gcm.encrypt(key, iv, toEncryptBytes);
+    const cryptoKey = await gcm.importKey(key);
```

Comment:
as @lanwen commented this needs to be part of this function. 

Usually this should have been a different PR. This PR should only take care of the contract tests, if there is no objections I will leave this change here, otherwise we can split it.